components/automate-cli/pkg/verifyserver/utils/checkutils/checkutil_test.go

```diff
@@ -44,9 +44,10 @@ func TestPrepareTriggerResponse_WithNoError(t *testing.T) {
 	check := "test check"
 	msg := "test message"
 	isError := false
+	statusCode := 200
 
 	// Act
-	result := PrepareTriggerResponse(resp, host, nodeType, errorString, check, msg, isError)
+	result := PrepareTriggerResponse(resp, host, nodeType, errorString, check, msg, isError, statusCode)
```

Comment:
```suggestion
	result := PrepareTriggerResponse(resp, host, nodeType, errorString, check, msg, isError, fiber.StatusOK)
```
you can directly use fiber.StatusOK